schemas/jetstream/api/v1/consumer_configuration.json

```diff
@@ -110,6 +110,7 @@
       ]
     },
     "ack_wait": {
+      "description": "How long (in nanoseconds) to allow messages to remain un-acknowledged before attempting redelivery",
```

Comment:
Thank you! do you mind updating here instead:

https://github.com/nats-io/jetstream/blob/2734c559e8090f59be4e0995f14b266935228c39/schema_source/jetstream/api/v1/definitions.json#L288

then when I regenerate the schemas from source all the related one get the update :)

Reply:
👍 not at all. I changed that.